packages/tectonic-explorer/src/components/planet-view.global.scss

```diff
@@ -1,18 +1,18 @@
-@import "variables.less";
+@import "./variables.scss";
 
 .planet-view {
   position: relative;
 
   &.full {
-    // ~"..." is a way to escape Less processing. Otherwise, Less compiles it to calc(40%)...
+    // unquote("...") is a way to escape Less processing. Otherwise, Less compiles it to calc(40%)...
```

Comment:
Should change to:  `// unquote("...") is a way to escape SASS processing. Otherwise, SASS compiles it to calc(40%)...`

Reply:
Thanks, I just fixed it in the last commit (and moved to a better place actually).